benchmark-scripts/Makefile

```diff
@@ -16,5 +16,13 @@ python-test:
 python-integration:
 	python -m coverage run -m unittest benchmark_integration.py
 
-python-coverage:
-	coverage report -m
\ No newline at end of file
+# for more up-to-date coverage, run python unit test first
+python-coverage: python-test
+	coverage report -m
+
+# this one is just used for easier deployment of changes on benchmark-scripts
+# to loss-prevention LOCALLY without reaching out to github
+# assuming loss-prevention/ repo is already git cloned before and is
+# in the same parallel directory structure as performance-tools/ repo
+copy-to-loss-prevention:
```

Comment:
Performance tools are a submodule of loss-prevention and can be modified to point to a specific git hash if needed.

Reply:
One recommendation is to show users through documentation how to point the benchmark script to 2 different docker-compose files (ex. Automated self checkout & loss prevention)

Reply:
this one is just for local development without the latest hash, so no internet needed

Reply:
> One recommendation is to show users through documentation how to point the benchmark script to 2 different docker-compose files (ex. Automated self checkout & loss prevention)

currently i am doing this for two separate pipelines or services in one docker-compose file. we can expand it to multiple compose file in future as I understand the requirements are more aligned for the case i have so far (like loss-prevent but use two different pipelines, one for object detection with roi and one for business logic roi handling pipeline for example and I assume they are in the same docker compose file since that is the deployment unit)

Reply:
and also i will add the example in another PR in the draft loss-prevent PR i have so far to show case this

Reply:
The new docker compose file looks good and makes sense for the current use case.

Reply:
Having offline mode is not required for this particular use case. If we did want to create a local copy it might be easier to do this from the use-case repo. This avoid having a circular dependency between repos and sub repos

Reply:
sure, i can remove this make target; this one just for local development without worry about git merging into the main branch first for performance-tool those type of stuff because loss-prevention does use this as library for dependency.